bootstrap/tests/test_raw_updater.py

```
@@ -0,0 +1,428 @@
+        assert result is None
+
+
+class TestUploader:
+    def test_resolve_storage_without_overrides(self) -> None:
+        config = DeveloperCiRawArtifacts(remote_root="data-generator/raw-artifact")
+        storage = DeveloperCiStorage(
+            endpoint="https://example.com",
+            bucket="bucket",
+            alias="alias",
+            access_key="key",
+            secret_key="secret",
```

Ruff (0.15.20)

[error] Possible hardcoded password assigned to argument: "secret_key"

(S106)